scripts/estimate.ts

```diff
@@ -0,0 +1,30 @@
+import { promises as filesystem } from 'fs'
+import * as path from 'path'
+import { CompilerOutput, CompilerInput, compileStandardWrapper, CompilerOutputContract } from 'solc'
+import { ethers } from 'ethers'
+import dotenv from "dotenv";
+import yargs from 'yargs/yargs';
+import { compileContracts } from './utils';
+
+dotenv.config()
+
+async function doStuff() {
+	const rpcUrl = process.env.RPC
+	const provider = new ethers.providers.JsonRpcProvider(rpcUrl)
```

Comment:
Is this the `script to get tx params`? It's not really clear to me what this PR is addressing